apps/marketplace/components/SellerSelect/SellerSelect.js

```diff
@@ -0,0 +1,134 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import AUtextInput from '@gov.au/text-inputs/lib/js/react.js'
+import AUbutton from '@gov.au/buttons/lib/js/react.js'
+import findSuppliers from 'marketplace/actions/supplierActions'
+import styles from './SellerSelect.scss'
+
+const SellerSelectView = props => (
+  <div>
+    <label htmlFor={props.id}>Select sellers</label>
+    <AUtextInput
+      id={props.id}
+      placeholder={props.placeholder}
+      onChange={props.handleOnChange}
+      value={props.inputValue}
+      className={props.className}
+    />
+    {props.showSearchButton && <AUbutton onClick={props.handleSearchClick}>Search</AUbutton>}
+  </div>
+)
+
+const SellerSelectResultsView = props => (
+  <ul className={props.className}>
+    {props.sellers.map(seller => (
+      <li key={seller.code}>
+        <a href={`#${seller.code}`} onClick={e => props.handleSellerSelectClick(seller, e)}>
+          {seller.name}
+        </a>
+        {seller.panel && <span className={props.panelBadgeClassName}>✓ Panel</span>}
```

Comment:
The check mark may need to be in a separate `<span>` so that screen readers ignore it.  See https://www.w3.org/WAI/GL/wiki/Unicode_Character_with_an_On-Screen_Text_Alternative.  Might be a question for a11y team?